README.md

```diff
@@ -466,40 +466,45 @@ PropertyTestEntity::PropertyTestEntity()
 ##### Local references
-Local reference systems simply define affine transformations (currently only rotation and translation) between each other, every `LocalCS` having at most one parent coordinate system, which can be another `LocalCS` or a `ProjectionCS`.
+Local reference systems simply define affine transformations (currently only rotation and translation) between each other, every `LocalCS` having at most one parent coordinate system, which can be another `LocalCS` or a global cartesian like `ProjectionCS` and `GeocentricCS`.
 
 ##### Geometry transformation
 The `Geometry`-class implements methods to set the current reference systems as well as transforming it into another one. There are a few things that can go wrong during `Geometry::transformToCS`, so it may throw an exception on you. If it does, the geometry remains in a valid, unchanged state.
 
 There exists some computational overhead for the transformations: First of all, the transformation is computed whenever it is needed, which means that the tree of reference frames is traversed from the source and target systems to their respective roots -- which shouldn't be too expensive. After that, there are two possibilities:
 
 1. The root coordinate systems are the same. In this case, one full transformation can be computed (source --> root --> target) and applied to the geometry in one iteration over its coordinates.
-2. The root coordinate systems are different. Since GDAL takes care of the transformation between them, and the local systems are an addition of sempr, we need three steps to transform the geometry: 
+2. The root coordinate systems are different. In this case we need three steps to transform the geometry: 
 	1. Transform from source local to source root
-	2. Let GDAL transform it from source root to target root
+	2. Apply a global transformation as `geos::geom::CoordinateSequenceFilter`
 	3. Transform from target root to target
 
 
 #### Serialization
-The `traits-sqlite-geometry.hxx` implements a traits-class for `OGRGeometry*` with templated methods to store any pointer to an `OGRGeometry` or derived class. Currently, only a selected subset of GDALs geometries is supported, but serialization support is easily extended by inheriting from this traits-class: E.g., the line:
+The `traits-sqlite-geom-geometry.hxx` implements a traits-class for `geos::geom::Geometry*` with templated methods to store any pointer to an `geos::geom::Geometry` or derived class. The serialization support is easily extended by inheriting from this traits-class: E.g., the line:
 
 ```c++
-template <> class value_traits<OGRPolygon*, id_blob> : public value_traits<OGRGeometry*, id_blob> {};
+template <> class value_traits<geom::Point*, id_text> : public value_traits<geom::Geometry*, id_text> {};
```

Comment:
You changed the second template parameter from `id_blob` to `id_text`, but the next sentence still says:

>  enables serialization of `geos::geom::MultiPoint*` in ***binary*** form (WKB). In your entities, use e.g.: 

Which is now wrong.

Reply:
Copy and Past issue, sorry :-(